ui/src/index.js

```diff
@@ -41,7 +42,8 @@ async function main() {
     .register(AppKeyVis)
     .register(AppHome)
     .register(AppDemo)
-    .register(AppStatement);
+    .register(AppStatement)
+    .register(AppClusterinfo);
```

Comment:
Miss the `{this.renderAppMenuItem('clusterinfo')}` line in the layout/RootComponent.js